specifications/service-registry.yaml

```diff
@@ -0,0 +1,279 @@
+                  description: Location of the service catalog resource to register
+                  type: string
+                  format: catalogUri
+              required:
+                - uri
+      responses:
+        '200':
+          description: Successful operation
+        '403':
+          description: Catalog announcement forbidden
+        '405':
+          description: Invalid input
+  /catalogs/list:
+    get:
+      tags:
+        - catalogs
+      summary: List registered service catalogs
+      description: 
+        List service catalog URIs registered with this service registry. The 
+        query can be used by other Service Registries to mirror the content of
+        another registry.
+      operationId: listCatalogs
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+        '403':
+          description: Catalog listing forbidden
+  /services/find:
+    get:
+      tags:
+        - queries
+      summary: Query registered services
+      description: Receives a list of services fulfilling the specified query criteria.
+      operationId: findServices
+      parameters:
+        - in: query
+          name: lat
+          description: latitude of the user location (WGS84)
+          schema:
+            type: number
+        - in: query
+          name: lon
+          description: longitude of the user location (WGS84)
+          schema:
+            type: number
+        - in: query
+          name: radius
+          description: Radius around the coordinates to search in kilometers
+          schema:
+            type: number
+        - in: query
+          name: certs
+          description:
+            Return only service providers owning the specified certificate
+            in their certificate chain
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+        - in: query
+          name: types
+          description: Return only services matching the specified types
+          style: form
+          explode: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - route-gtfs
+                - route-netex
+                - routing-otp
+                - taxi-suti
+                - tickets-lippu
+                - tickets-bob
+                - tickets-ticketapi
+            uniqueItems: true
+        - in: query
+          name: sort
+          description: Sort service providers in specified order
+          schema:
+            type: string
+            uniqueItems: true
+            enum:
+              - rating
+              - popularity
+              - alpha
```

Comment:
Not clear what these mean, in particular, what's "alpha"?

Reply:
The idea of sort and limit parameters was to be able to limit the number of services returned by the query. Analogous to ORDER BY and LIMIT in sql. I updated the description, hopefully it is now more informative.